lib/daru/view.rb

```diff
@@ -14,7 +14,31 @@ def plotting_library= lib
         else
           raise ArgumentError, "Unsupported library #{lib}"
         end
-      end # def end
+      end
+
+      # Load the dependent JS files in IRuby notebook. Those JS will help in
+      # plotting the charts in IRuby cell.
+      #
+      # @example
+      #
+      # To load the dependent JS file for Nyaplot library
+      # plotting system (Nyaplot.js, d3.js):
+      #
+      # Daru::View.load_lib_in_iruby('Nyaplot')
+      #
+      # To load the HighCharts dependent JS
+      # files (highcharts.js, highcharts-3d.js, highstock.js):
+      #
+      # Daru::View.load_lib_in_iruby('HighCharts')
+      #
```

Comment:
Wouldn't it be great if you automate this i.e. the JS files are loaded automatically if needed?